### PR TITLE
libunwind rebuild without leaking ENV variables

### PR DESCRIPTION
**Pull request by satmandu**
Works properly:
- [x] x86_64

Builds properly:
- [x] x86_64
- [x] armv7l
- [x] i686

### CODE REVIEW

**Comment by satmandu** [comment]
(This was causing havoc with other compiles.)

File: packages/libunwind.rb [hunk 1]
```diff
@@ -3,33 +3,29 @@
...
-     armv7l: '96274d710810abd35f6a38f1c2d8c39f85959cc26de07f98a78dd27ff03dce9c',
-       i686: '508923a9dbdbacb0eeab2244db0c7caca2e8bcd3acd4d1c154c63eb600483937',
-     x86_64: '3453e7f9438a665df84f39fcbd1c996009988480f05ec3489712efdf3c1499e0',
+     aarch64: 'c1b315c17464f67a1e719a615da0154c661eaf1e6e1584f8a68870a2de8721e8',
+      armv7l: 'c1b315c17464f67a1e719a615da0154c661eaf1e6e1584f8a68870a2de8721e8',
+        i686: 'd697d0a6c015dc8c01593669679c367ca0c9f412e140adb2d4ced0af9fb42fb2',
+      x86_64: 'a759499690b70258e91d8ac7cd32047a3cf3308364b04f062e8fc60a33782ec6',
   })
 
   depends_on 'xzutils' => :build
```
Comment (uberhacker) [inline packages/libunwind.rb:24]: Core package?

Comment (satmandu) [reply packages/libunwind.rb:24]: Yup.  I'll update install.sh

Comment (satmandu) [reply packages/libunwind.rb:24]: I misunderstood what you were saying. You're right. Removing xzutils.

**Pull request merged**